### PR TITLE
Use yum repo module instead of echoing to a file

### PR DESCRIPTION
**Pull request by vishesh92**

### CODE REVIEW

**Review by sureshanaparti — APPROVED** [review]

File: Ansible/tasks/set_centos_repos.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+
+- name: Cleanup centos repo files
```
Comment (sureshanaparti) [inline Ansible/tasks/set_centos_repos.yml:3]:
```suggestion
- name: Set yum repo to local
```

Comment (sureshanaparti) [reply Ansible/tasks/set_centos_repos.yml:3]: Ignore  this, as it removes the repo files

**Comment by sureshanaparti** [comment]
Verified MS, KVM host and Marvin box in CentOS 7 based env.

File: Ansible/tasks/set_centos_repos.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    path: "{{ item }}"
+    state: absent
+  with_items:
+    - /etc/yum.repos.d/epel.repo
+    - /etc/yum.repos.d/CentOS-Base.repo
+
+- name: Add epel repo
+  yum_repository:
+    name: epel
+    description: Extra Packages for Enterprise Linux 7 - $basearch
```
Comment (vishesh92) [inline Ansible/tasks/set_centos_repos.yml:14]:
```suggestion
    description: Extra Packages for Enterprise Linux - $basearch
```

**Pull request merged**